portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,10 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+body {
+
```

Comment:
remove the extra lines before and after "background-color"
body {
   background-color .... 
}